README.md

```diff
@@ -76,4 +77,16 @@ By default, installing a package with the `—save` option, npm saves the packag
 npm config set save-exact
 ```
 
+[↑ back to Table of Contents](#table-of-contents)
+
+## Avoid relying on globally installed packages
```

Comment:
rephrase: "Avoid installing modules globally"

summary (needs better phrasing): "NPM first tries globally installed modules before looking for local modules. Globally installed modules are shared between projects but might not match the expected version.".